src/fonts-context.js

```diff
@@ -0,0 +1,25 @@
+import { useState, createContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+export const ManageFontsContext = createContext();
+export const DEFAULT_DEMO_TEXT = __( "The quick brown fox jumps over the lazy dog.", "create-block-theme" );
+
+
+export function ManageFontsProvider( { children } ) {
+    const [ demoText, setDemoText ] = useState( localStorage.getItem( "create-block-theme_default-demo-text" ) || DEFAULT_DEMO_TEXT );
+
+    const handleDemoTextChange = ( newDemoText ) => {
+        setDemoText( newDemoText );
+        localStorage.setItem( "create-block-theme_default-demo-text", newDemoText );
```

Comment:
Is the benefit of using localStorage that the custom text will be preserved in the browser when the user closes + navigates away from this page?

Reply:
> Yes, and also this is not a pure Single Page Application so we need to store the data somewhere when the page reloads.